tasklist/pom.xml

```diff
@@ -25,7 +25,6 @@
     <module>importer</module>
     <module>archiver</module>
     <module>webapp</module>
-    <module>qa</module>
     <module>../tasklist-distro</module>
     <module>mvc-auth-commons</module>
     <module>test-coverage</module>
```

Comment:
can you also move `test-coverage` to integration-tests profile, as it depends on integration-tests module

Reply:
Sure 👍🏼